main.go

```diff
@@ -77,8 +78,7 @@ func (vs *VideoStream) Close() error {
 // user and the requested resource.  this bandwidth is computed by downloading up to 10MB.
 func (vs *VideoStream) bandwidth() (float64, error) {
 	tbefore := time.Now()
-	buf := make([]byte, 10000000)
-	n, err := io.ReadFull(vs.tee, buf)
+	n, err := io.Copy(ioutil.Discard, vs.tee)
```

Comment:
This will copy the entire file, but we only want to download a small portion to measure bandwidth. I think the best thing to do here is declare a constant, `bandwidthSampleSize`, then use `io.CopyN`:

```
// bandwidthSampleSize is the number of bytes to download in order to determine the available download bandwidth.
const bandwidthSampleSize = 10000000`
```
..... in bandwidth():
```
n, err := io.CopyN(ioutil,Discard, vs.tee, bandwidthSampleSize)
```

Thanks for the PR.